libafl_bolts/src/os/mod.rs

```diff
@@ -115,6 +115,28 @@ pub fn dup(fd: RawFd) -> Result<RawFd, Error> {
     }
 }
 
+// Derived from https://github.com/RustPython/RustPython/blob/7996a10116681e9f85eda03413d5011b805e577f/stdlib/src/resource.rs#L113
+// LICENSE: MIT https://github.com/RustPython/RustPython/commit/37355d612a451fba7fef8f13a1b9fdd51310b37e
+/// Get the peak rss (Resident Set Size) of the all child processes 
+/// that have terminated and been waited for
+#[cfg(all(unix, feature = "std"))]
+pub fn peak_rss_mb() -> Result<i64, Error> {
```

Comment:
we should probably calle it `_children` or something, right? Or does it also check the own process?

Reply:
No, only the children so we can prepend _children

Reply:
Only the children so yeah we can prepend ``_children``